docs/frameworks/dapr.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Use Dapr with .NET Aspire
+description: Learn how to use Dapr with .NET Aspire
+ms.date: 05/14/2024
+ms.topic: overview
+---
+
+# Use Dapr with .NET Aspire
+
+[Distributed Application Runtime (Dapr)](https://docs.dapr.io/) offers developer APIs that run as a sidecar process and abstract away the common complexities of the underlying cloud platform. Dapr and .NET Aspire work together to improve your local development experience. By using Dapr with .NET Aspire, you can focus on writing and implementing .NET-based distributed applications instead of spending extra time with local onboarding.  
+
+In this guide, you'll learn how to take advantage of Dapr's abstraction and .NET Aspire's opinionated configuration of cloud technologies to build simple, portable, resilient, and secured microservices at-scale on Azure.
+
+[!INCLUDE [aspire-prereqs](../includes/aspire-prereqs.md)]
+
+In addition to the prerequisites for .NET Aspire, you will need:
+
+- Dapr version 1.13 or later
+
+To install Dapr, see [Install the Dapr CLI](https://docs.dapr.io/getting-started/install-dapr-cli/). After installing the Dapr CLI, run the `dapr init` described in [Initialize Dapr in your local environment](https://docs.dapr.io/getting-started/install-dapr-selfhost/).
+
+## Get started
+
+To get started you need to add the Dapr hosting package to your app host project by installing the [Aspire.Hosting.Dapr](https://www.nuget.org/packages/Aspire.Hosting.Dapr) NuGet package.
+
+### [.NET CLI](#tab/dotnet-cli)
+
+```dotnetcli
+dotnet add package Aspire.Hosting.Dapr
```

Comment:
At least prior to GA, you'll need the prerelease flag too, I think?

Reply:
@IEvangelist will this be published once this is GA?

Reply:
Yes, this goes live tomorrow morning, just before `//build`.